client/components/manageableProduct.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react'
+import ProductForm from './productForm'
+import {connect} from 'react-redux'
+import {deleteProductById} from '../store/product'
+
+class ManageableProduct extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      edit: false
+    }
+  }
+
+  handleSelectSubmit = (e, id) => {
+    e.preventDefault()
+    if (e.target.value === 'delete') {
+      this.props.deleteProduct(id)
+      //reroute to AllProducts
```

Comment:
Make this a `// TODO: ...` comment for better editor tooling support and clarity